src/main/java/org/candlepin/service/model/EventType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2009 - 2024 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.service.model;
+
+/**
+ * An enum that defines the types of Candlepin {@link Event}s.
+ */
+public enum EventType {
+
+    /**
+     * An event where a cloud system has checked-in.
```

Comment:
It would be good to describe what exactly is a check-in in this context (that the Consumer.lastCheckin timestamp was updated).

Reply:
I agree! Updated